app/services/concerns/xccdf/hosts.rb

```diff
@@ -20,9 +20,8 @@ def report_host
     private
 
     def test_result_profile
-      @profiles.find do |profile|
-        profile.ref_id == @test_result_file.test_result.profile_id
-      end
+      ::Profile.find_by(ref_id: @test_result_file.test_result.profile_id,
```

Comment:
This is unrelated, right? It makes sure the profile found is the "template" profile. Maybe we should have a helper for that in the model

Reply:
This isn't unrelated. If we don't save the benchmark, `@profiles` never gets set. It's no problem, though, since this method just selects the profile that is referenced in the `TestResult`. This change just changes from selecting from the existing array of profiles from the benchmark parsing code to querying directly from the database. Now it works for both parsing benchmarks and skipping benchmark parsing.